### PR TITLE
fix: returns the healthcheck command in an array of string

### PR DESCRIPTION
**Pull request by seongm-1**
Address #1079 

## svc logs

image

## aws console

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by bvtujo** [comment]
Could you show some log output from a sample deployment here, indicating that it executes correctly and doesn't cause flapping containers?

File: internal/pkg/docker/dockerfile/dockerfile.go [hunk 1]
```diff
@@ -248,12 +249,17 @@ func parseHealthCheck(content string) (*HealthCheck, error) {
 		return nil, err
 	}
 
+	// cmdIndex holds the starting and ending index of "CMD"
+	// command extracts the CMD out of the command
+	cmdIndex := regexp.MustCompile("CMD").FindStringIndex(content)
```
[the remark this answers is not in the document]
Comment (efekarakus) [reply internal/pkg/docker/dockerfile/dockerfile.go:254]: Can we add a comment saying that if the HEALTHCHECK instruction is not "NONE" then there must be a "CMD" instruction.

File: internal/pkg/docker/dockerfile/dockerfile.go [hunk 2]
```diff
@@ -249,10 +250,11 @@ func parseHealthCheck(content string) (*HealthCheck, error) {
 		return nil, err
 	}
 
-	// cmdIndex holds the starting and ending index of "CMD"
+	// cmdIndex holds the starting index of "CMD", -1 if not present
```
Comment (efekarakus) [inline internal/pkg/docker/dockerfile/dockerfile.go:253]: This comment is not needed since it can't be -1

Comment (seongm-1) [reply internal/pkg/docker/dockerfile/dockerfile.go:253]: Ah good catch, I forgot to take that out while testing

**Pull request merged**